package.json

```diff
@@ -41,13 +41,16 @@
     "@types/is-base64": "^1.1.0",
     "@types/jest": "^26.0.23",
     "@types/lodash": "latest",
+    "@types/react-router-dom": "5",
     "@types/valid-url": "^1.0.3",
     "eslint-plugin-import": "^2.23.4",
     "eslint-plugin-jest-dom": "^3.9.0",
     "eslint-plugin-react-hooks": "^4.2.0",
     "eslint-plugin-testing-library": "^4.10.1",
     "husky": "^6.0.0",
     "jest-environment-jsdom-fifteen": "^1.0.2",
+    "react-router": "^5.2.0",
+    "react-router-dom": "^5.2.0",
```

Comment:
Do we still need these installed locally? I forgot

Reply:
Good call! We don't.